examples/ffd/README.rst

```diff
@@ -0,0 +1,174 @@
+
+See the full documentation for more information on configuring, modifying, building, and running the firmware.
+
+Speech Recognition
+******************
+
+The application will recognize the following utterances:
+
+Wakeword Utterances
+-------------------
+
+- Hello XMOS
+- Hello Cyberon (DSpotter™ model only)
+
+Command Utterances
+------------------
+
+- Switch on the TV
+- Channel up
+- Channel down
+- Volume up
+- Volume down
+- Switch off the TV
+- Switch on the lights
+- Brightness up
+- Brightness down
+- Switch off the lights
+- Switch on the fan
+- Speed up the fan
+- Slow down the fan
+- Set higher temperature
+- Set lower temperature
+- Switch off the fan
+
+Supported Hardware and pre-requisites
+=====================================
+
+This example is supported on the XK_VOICE_L71 board.
+
+On the host machine the XTC tools, version 15.2.1, must be installed and sourced.
+The output should be
+something like this:
+
+::
+
+   $ xcc --version
+   xcc: Build 19-198606c, Oct-25-2022
+   XTC version: 15.2.1
+   Copyright (C) XMOS Limited 2008-2021. All Rights Reserved.
+
+On Windows it is highly recommended to use ``Ninja`` as the make system under
+``cmake``. Not only is it a lot faster than MSVC ``nmake``, it also
+works around an issue where certain path names may cause an issue with
+the XMOS compiler under Windows.
+
+To install Ninja, follow these steps:
+
+-  Download ``ninja.exe`` from
+   https://github.com/ninja-build/ninja/releases. This firmware has been
+   tested with Ninja version v1.11.1.
+-  Ensure Ninja is on the command line path. It can be added to the path
+   permanently by following these steps
+   https://www.computerhope.com/issues/ch000549.htm. Alternatively,
+   set the path in the current command line session using something
+   like ``set PATH=%PATH%;C:\Users\xmos\utils\ninja``
+
+Before building the host application, you will need to add the path to the XTC Tools to your environment.
+
+  set "XMOS_TOOL_PATH=<path-to-xtc-tools>"
+
+Building the Host Applications
+==============================
+
+This application requires a host application to create the flash data partition. Run the following commands in the root folder to build the host application using your native Toolchain:
+
+.. note::
+
+    In the commands below ``<speech_engine>`` can be either ``sensory`` or ``cyberon``, depending on the choice of the speech recognition engine and model.
+
+.. note::
+
+    Permissions may be required to install the host applications.
+
+On Linux or Mac run:
+
+::
+
+    cmake -B build_host
+    cd build_host
+    make install
+
+The host applications will be installed at ``/opt/xmos/bin``, and may be moved if desired.  You may wish to add this directory to your ``PATH`` variable.
+
+Then build the host application:
+
+::
+
+    cmake -G Ninja -B build_host
```

Comment:
Ah - you have already done this. I need to re-work my PR https://github.com/xmos/sln_voice/pull/333/files

Reply:
Yes, but I forgot to replace `CMAKE_TOOLCHAIN_FILE`.